rootio/tchain_test.go

```diff
@@ -292,9 +292,20 @@ func TestSeekEntry(t *testing.T) {
 
 	var entries = []int64{1, 0, 2, 9, 10, 11, 0, 19}
 	for _, entry := range entries {
-		err := sc.SeekEntry(entry)
+		err = sc.SeekEntry(entry)
 		if err != nil {
-			t.Error("Could not seek entry %d", entry)
+			t.Fatal(err)
 		}
+		for sc.Next() {
```

Comment:
this will loop over the whole remaining entries in the file.
you do have to call `Next()` to load the data associated with the current entry, though.
so:
```go
if !sc.Next() {
    // handle error
    ...
}
```

Reply:
i keep having different values when i compare float64(entry) and F64 field,
please, is there something else that needs a correction ?

Reply:
what are those?

Reply:
It kinds of work, but : 
If for example the slice of entries is {1, 2, 14, 13}, ie : the transition from one tree to another is not via the first element of that tree, then it doesn't work. (this will work as an example : {2, 4, 7, 10, 15, 0}

Reply:
kind of works*